src/workflow_docs/fetch_docs_sources.sh

```diff
@@ -16,18 +16,26 @@ git clone --no-checkout --depth=1 --single-branch --branch=master --filter=tree:
+git clone --no-checkout --depth=1 --single-branch --branch=main   --filter=tree:0 https://github.com/microbiomedata/metaT_ReadsQC /tmp/clones/metaT_ReadCounts
 git clone --no-checkout --depth=1 --single-branch --branch=master --filter=tree:0 https://github.com/microbiomedata/metaPro /tmp/clones/metaPro                     && cd /tmp/clones/metaPro           && git sparse-checkout set --no-cone /docs && git checkout
 git clone --no-checkout --depth=1 --single-branch --branch=master --filter=tree:0 https://github.com/microbiomedata/metaMS /tmp/clones/metaMS                       && cd /tmp/clones/metaMS            && git sparse-checkout set --no-cone /docs && git checkout
 git clone --no-checkout --depth=1 --single-branch --branch=master --filter=tree:0 https://github.com/microbiomedata/enviroMS /tmp/clones/enviroMS                   && cd /tmp/clones/enviroMS          && git sparse-checkout set --no-cone /docs && git checkout
 
 # Arrange the "docs" directories into chapters.
 mkdir -p /tmp/book/src/chapters
-mv /tmp/clones/ReadsQC/docs           /tmp/book/src/chapters/1_RQC
-mv /tmp/clones/ReadbasedAnalysis/docs /tmp/book/src/chapters/2_ReadAnalysis
-mv /tmp/clones/metaAssembly/docs      /tmp/book/src/chapters/3_MetaGAssembly
-mv /tmp/clones/mg_annotation/docs     /tmp/book/src/chapters/4_MetaGAnnotation
-mv /tmp/clones/metaMAGs/docs          /tmp/book/src/chapters/5_MAG
-mv /tmp/clones/metaT/docs             /tmp/book/src/chapters/6_MetaT
-mv /tmp/clones/metaPro/docs           /tmp/book/src/chapters/7_Metaproteomics
-mv /tmp/clones/metaMS/docs            /tmp/book/src/chapters/8_Metabolomics
-mv /tmp/clones/enviroMS/docs          /tmp/book/src/chapters/9_NOM
+#placeholder for metaG end to end chapter which will come from this repo
+#                                      /tmp/book/src/chapters/1_Metagenome Workflow Overview
+mv /tmp/clones/ReadbasedAnalysis/docs /tmp/book/src/chapters/2_Read Based Taxonomy 
+mv /tmp/clones/ReadsQC/docs           /tmp/book/src/chapters/3_Metagenome Reads QC 
+mv /tmp/clones/metaAssembly/docs      /tmp/book/src/chapters/4_Metagenome Assembly
+mv /tmp/clones/mg_annotation/docs     /tmp/book/src/chapters/5_Metagenome and Metatranscriptome Annotation
+mv /tmp/clones/metaMAGs/docs          /tmp/book/src/chapters/6_Metagenome Assembled Genome 
+mv /tmp/clones/metaT/docs             /tmp/book/src/chapters/7_Metatranscriptome Worflow Overview
+mv /tmp/clones/metaT_ReadsQC/docs     /tmp/book/src/chapters/8_Metranscriptome Reads QC 
+mv /tmp/clones/metaT_Assembly/docs     /tmp/book/src/chapters/9_Metranscriptome Assembly
+mv /tmp/clones/metaT_ReadCounts/docs     /tmp/book/src/chapters/10_Metranscriptome Expression 
+mv /tmp/clones/metaPro/docs           /tmp/book/src/chapters/11_Metaproteomics
+mv /tmp/clones/metaMS/docs            /tmp/book/src/chapters/12_Metabolomics
+mv /tmp/clones/enviroMS/docs          /tmp/book/src/chapters/13_Natural Organic Matter
```

Comment:
I'd prefer all the things on the right use underscores (or hyphens) _instead of spaces_. Those strings are going to be part of the chapters' URLs, and spaces in URLs can add complexity to certain things (e.g. having to URL-[encode](https://en.wikipedia.org/wiki/Percent-encoding#Character_data) the spaces to `%20` in certain situations).

Reply:
For example (here, I've replaced the spaces with underscores):

```suggestion
mv /tmp/clones/ReadbasedAnalysis/docs /tmp/book/src/chapters/2_Read_Based_Taxonomy 
mv /tmp/clones/ReadsQC/docs           /tmp/book/src/chapters/3_Metagenome_Reads_QC 
mv /tmp/clones/metaAssembly/docs      /tmp/book/src/chapters/4_Metagenome_Assembly
mv /tmp/clones/mg_annotation/docs     /tmp/book/src/chapters/5_Metagenome_and_Metatranscriptome_Annotation
mv /tmp/clones/metaMAGs/docs          /tmp/book/src/chapters/6_Metagenome_Assembled_Genome 
mv /tmp/clones/metaT/docs             /tmp/book/src/chapters/7_Metatranscriptome_Worflow_Overview
mv /tmp/clones/metaT_ReadsQC/docs     /tmp/book/src/chapters/8_Metranscriptome_Reads_QC
mv /tmp/clones/metaT_Assembly/docs    /tmp/book/src/chapters/9_Metranscriptome_Assembly
mv /tmp/clones/metaT_ReadCounts/docs  /tmp/book/src/chapters/10_Metranscriptome_Expression
mv /tmp/clones/metaPro/docs           /tmp/book/src/chapters/11_Metaproteomics
mv /tmp/clones/metaMS/docs            /tmp/book/src/chapters/12_Metabolomics
mv /tmp/clones/enviroMS/docs          /tmp/book/src/chapters/13_Natural_Organic_Matter
```

Also, I think there are a couple typos:
- In 7_...: "Worflow" versus "Workflow"
- In 10_...: "met" versus "meta"